### DISCUSSION TITLE
是否能搭配cloudflare一起使用

### DISCUSSION THREAD

**Discussion by money-outlook** [apernet/hysteria#401] [Q&A]
能否像v2ray 的ws+tls那樣，配合cloudflare cdn代理使用，讓客戶端連cloudflare cdn，再由該cdn反代vps？

自己試了下好像不行，特請教。

本人配置：
cloudflare開啟代理，開啟quic支持，走的是8443的端口（cloudflare可對等代理8443、443等端口，在ws+tls下測試可用）

hysteria開obfs、配置的alpn字段填寫"h3"、"server"字段填寫"域名:8443"，位於vps的hysteria服務端監聽的是ipv6地址

以上配置無法使用，提示網路不可用

**Comment by haruue** [comment]
不能。 
如果你要配合 Cloudflare 使用， 请用 v2fly+h2/ws+vmess/vless。
问这个问题的人实在太多了。 我们会考虑专门开一个 wiki 页面来阐述这个问题。

UPDATE: https://hysteria.network/zh/docs/misc/CDN/

**Reply by ghost** [reply]
那如果是ip被封了，是不是就用不了这个了？还想着走域名抢救一下
